book/src/install.md

```diff
@@ -34,7 +36,7 @@ brew install axodotdev/tap/cargo-dist
 
 #### Pacman (Arch Linux)
 
-Arch Linux users can install `cargo-dist` from the [extra repository](https://archlinux.org/packages/extra/x86_64/cargo-dist/) using [pacman](https://wiki.archlinux.org/title/Pacman):
```

Comment:
we may need to leave this one as the arch package is not renamed

Reply:
Oh, good call. I intended this to be "you can install `dist` (the software) using the command bellow (with a different name)" but I realize there's a lot of ambiguity in how this is written.